README.md

```diff
@@ -22,22 +22,20 @@
 <p align="center">Data Labeling + Data Curation + Inference Store</p>
 <p align="center">Designed for MLOps & Feedback Loops</p>
 
-
```

Comment:
I see some fixes regarding old URL links. Could we have a separate PR for these changes so we can easily move changes to both, the `develop` and `main` branches?